lib/client/conntest/externalcloudaudit.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conntest
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth"
+)
+
+// ExternalCloudAuditConnectionTesterConfig defines the config fields for ExternalCloudAuditConnectionTester.
+type ExternalCloudAuditConnectionTesterConfig struct {
+	// UserClient is an auth client that has a User's identity.
+	UserClient auth.ClientI
+}
+
+// ExternalCloudAuditConnectionTester implements the ConnectionTester interface for Testing ExternalCloudAudit access.
+type ExternalCloudAuditConnectionTester struct {
+	cfg ExternalCloudAuditConnectionTesterConfig
+}
+
+// NewDatabaseConnectionTester returns a new DatabaseConnectionTester.
+func NewExternalCloudAuditConnectionTester(cfg ExternalCloudAuditConnectionTesterConfig) (*ExternalCloudAuditConnectionTester, error) {
+	return &ExternalCloudAuditConnectionTester{
+		cfg,
+	}, nil
+}
+
+// TestConnection tests the current configured externalcloudaudit draft by:
+// * Uploading a
```

Comment:
comment should be completed

Reply:
Fixed in [9cb97c8](https://github.com/gravitational/teleport/pull/34278/commits/9cb97c897849648a6cb93f7fe25ec926f07b0b11)